lib/ws-kernel-picker.js

```diff
@@ -153,11 +170,13 @@ export default class WSKernelPicker {
           options
         };
       });
-      kernelListing.setItems(items);
+      kernelListing.selectListView.update({ items: items });
+      kernelListing.selectListView.focus();
+      kernelListing.selectListView.reset();
```

Comment:
In my other recent PR's the picker classes I used had `toggle` and `attach` methods that would do update/focus/resetting for us. I didn't add these methods this time to keep the class design similar to how it worked before, with most of the important stuff right in the constructor.